Library/Format.swift

```diff
@@ -144,6 +144,12 @@ public enum Format {
     return NSAttributedString(string: Strings.plus_shipping_cost(shipping_cost: ""), attributes: attributes)
   }
 
+  public static func attributedAmount(
```

Comment:
I created this function to return the attributed amount without the `+` sign

Reply:
I can't think of a reason that we'd want to do this 🤔

Reply:
We can probably remove this function now, right?